articles/cognitive-services/Face/QuickStarts/Python.md

```diff
@@ -4,317 +4,141 @@ description: Get information and code samples to help you quickly get started us
 services: cognitive-services
 author: v-royhar
 manager: yutkuo
-
 ms.service: cognitive-services
 ms.technology: face
 ms.topic: article
 ms.date: 06/21/2017
 ms.author: anroth
 ---
 
-# Face API Python Quick Starts
-This article provides information and code samples to help you quickly get started using the Face API with Python to accomplish the following tasks: 
-* [Detect Faces in Images](#Detect) 
-* [Create a Person Group](#Create)
+# Face detection using Cognitive Services
+In this walkthrough, we will the cognitive services [Face APIs](https://azure.microsoft.com/en-us/services/cognitive-services/face/) to detect faces in an image and identify various attributes such as the number of people, their gender and age. The sample images we will use are from the [How-Old Robot](http://www.how-old.net) that uses the same service behind the scenes.
```

Comment:
"identify various attributes" is misleading considering the `Identification` operation we provided.